lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVM.cpp

```diff
@@ -1468,6 +1473,15 @@ Value ReduceOpConversion::shflSync(ConversionPatternRewriter &rewriter,
     return bitcast(vec, val.getType());
   }
 
+#ifdef USE_ROCM
+  DenseMap<short, unsigned int> masks{{16, 0x401F}, {8, 0x201F}, {4, 0x101F}, {2, 0x081F}, {1, 0x041F}};
```

Comment:
Please add comments on the meaning of the mask values.

Reply:
An explanation is added.